### PR TITLE
Added the icon in the tab and in the header 

### PR DESCRIPTION
**Pull request by JackyTu99**
I added the MobiStudy icon in the tab and in the header. I also replaced the quasar favicons with mobistudy's favicons. #36

### CODE REVIEW

**Review by dariosalvi78 — COMMENTED** [review]
please reinsert searchMedicationConcept and searchDiseaseConcept back

File: src/modules/API.js [hunk 1]
```diff
@@ -25,12 +25,6 @@ export default {
   async createUser (newuser) {
     return axios.post(BASE_URL + '/users', newuser)
   },
-  async searchDiseaseConcept (disease, lang) {
```
[the remark this answers is not in the document]
Comment (JackyTu99) [reply src/modules/API.js:28]: These where the two lines we had to remove. It was in our notes from the code review meeting but I can add them again
Capture

[the remark this answers is not in the document]
Comment (dariosalvi78) [reply src/modules/API.js:28]: got it, must have been a misunderstanding. Keep those two lines please.

[the remark this answers is not in the document]
Comment (JackyTu99) [reply src/modules/API.js:28]: will do!

**Review by dariosalvi78 — APPROVED** [review]

**Pull request merged**